bot.py

```diff
@@ -335,86 +338,88 @@ async def verify(ctx):
+            ('🇦🇹', "at", "Austria", ["German"]),
+            ('🇦🇿', "az", "Azerbeijan", ["Azerbaijani"]),
+            ('🇧🇾', "by", "Belarus", ["Belarusian", "Russian"]),
+            ('🇧🇪', "be", "Belgium", ["Dutch", "French", "German"]),
+            ('🇧🇦', "ba", "Bosnia & Herzegovina", ["Bosnian"]),
+            ('🇧🇬', "bg", "Bulgaria", ["Bulgarian"]),
+            ('🇭🇷', "hr", "Croatia", ["Croatian"]),
+            ('🇨🇾', "cy", "Cyprus", ["Greek", "Turkish"]),
+            ('🇨🇿', "cz", "Czechia", ["Czech"]),
+            ('🇩🇰', "dk", "Denmark", ["Danish"]),
+            ('🇪🇪', "ee", "Estonia", ["Estonian"]),
+            ('🇫🇮', "fi", "Finland", ["Finnish", "Swedish"]),
+            ('🇫🇷', "fr", "France", ["French"]),
+            ('🇩🇪', "de", "Germany", ["German"]),
+            ('🇬🇪', "ge", "Georgia", ["Georgian"]),
+            ('🇬🇷', "gr", "Greece", ["Greek"]),
+            ('🇭🇺', "hu", "Hungary", ["Hungarian"]),
+            ('🇮🇸', "is", "Iceland", ["Icelandic"]),
+            ('🇮🇪', "ie", "Ireland", ["Irish"]),
+            ('🇮🇹', "it", "Italy", ["Italian"]),
+            ('🇽🇰', "xk", "Kosovo", ["Kosovar"]),
+            ('🇰🇿', "kz", "Kazakhstan", ["Kazakh"]),
+            ('🇱🇻', "lv", "Latvia", ["Latvian"]),
+            ('🇱🇮', "li", "Liechteinstein", ["German"]),
+            ('🇱🇹', "lt", "Lithuania", ["Lithuanian"]),
+            ('🇱🇺', "lu", "Luxembourg", ["Luxembourgish", "French", "German"]),
+            ('🇲🇹', "mt", "Malta", ["Maltese"]),
+            ('🇲🇩', "md", "Moldova", ["Romanian"]),
+            ('🇲🇨', "mc", "Monaco", ["French"]),
+            ('🇲🇪', "me", "Montenegro", ["Montenegrin"]),
+            ('🇳🇱', "nl", "Netherlands", ["Dutch"]),
+            ('🇲🇰', "mk", "North Macedonia", ["Macedonian", "Albanian"]),
+            ('🇳🇴', "no", "Norway", ["Norwegian"]),
+            ('🇵🇱', "pl", "Poland", ["Polish"]),
+            ('🇵🇹', "pt", "Portugal", ["Portuguese"]),
+            ('🇷🇴', "ro", "Romania", ["Romanian"]),
+            ('🇷🇺', "ru", "Russia", ["Russian"]),
+            ('🇸🇲', "sm", "San Marino", ["Italian"]),
+            ('🇷🇸', "rs", "Serbia", ["Serbian"]),
+            ('🇸🇰', "sk", "Slovakia", ["Slovak"]),
+            ('🇸🇮', "si", "Slovenia", ["Slovenian"]),
+            ('🇪🇸', "es", "Spain", ["Spanish"]),
+            ('🇸🇪', "se", "Sweden", ["Swedish"]),
+            ('🇨🇭', "ch", "Switzerland", ["German", "French", "Italian"]),
+            ('🇹🇷', "tr", "Turkey", ["Turkish"]),
+            ('🇬🇧', "gb", "United Kingdom", []),
+            ('🇺🇦', "ua", "Ukraine", ["Ukrainian", "Russian"]),
+            ('🇻🇦', "va", "Vatican", ["Italian"]),
+            (':region_asia:', "asia", "Asia", []),
+            (':region_africa:', "africa", "Africa", []),
+            (':region_northamerica:', "us", "North America", []),
+            (':region_northamerica:', "ca", "North America", ["French"]),
+            (':region_northamerica:', 'mx', "North America", ["Spanish"]),
```

Comment:
the Bahamas, Bermuda, Canada, the Caribbean, Central America, Clipperton Island, Greenland, Mexico, Saint Pierre and Miquelon, Turks and Caicos Islands, and the United States. 

or maybe just consider doing northamerica